src/handlers/tests/interventions-processor-lambda.test.ts

```diff
@@ -74,94 +125,97 @@ describe('intervention processor handler', () => {
 
   describe('handle', () => {
     it('does nothing if SQS event contains no record', async () => {
-      mockEvent = { Records: [] };
       const loggerWarnSpy = jest.spyOn(logger, 'warn');
-      await handler(mockEvent, mockContext);
+      await handler({ Records: [] }, mockContext);
       expect(loggerWarnSpy).toHaveBeenCalledWith('Received no records.');
       expect(logAndPublishMetric).toHaveBeenCalledWith('INTERVENTION_EVENT_INVALID');
     });
 
-    it('should return a state transition error', async () => {
-      eventValidationMock.mockReturnValueOnce(void 0);
+    it('it should not retry the record if a StateTransitionError is received', async () => {
```

Comment:
absolute nitpick, sorry, can we remove the 'it' from the start of this? 
```suggestion
    it('should not retry the record if a StateTransitionError is received', async () => {
```